docs/FAQ.md

```diff
@@ -264,9 +264,6 @@ efficient to stay on the CPU instead of going back and forth.
 
 Yes, DPP still works.  It might not be as efficient as it could be, and we are working to improve it.
```

Comment:
@jlowe is this still a valid statement?  Can DPP be more efficient?

Reply:
The second sentence can be removed, we're no longer running the subquery twice as we once were.